src/Storages/Statistics/Statistics.cpp

```diff
@@ -171,6 +201,11 @@ MergeTreeStatisticsFactory::MergeTreeStatisticsFactory()
 
     registerValidator(StatisticsType::Uniq, UniqValidator);
     registerCreator(StatisticsType::Uniq, UniqCreator);
+
+#if USE_DATASKETCHES
+    registerValidator(StatisticsType::CountMinSketch, CountMinSketchValidator);
+    registerCreator(StatisticsType::CountMinSketch, CountMinSketchCreator);
```

Comment:
we can throw exception in `CountMinSketchCreator` when datasketches is not compiled.